pkg/generic/thrift/json.go

```diff
@@ -102,8 +102,8 @@ func (m *WriteJSON) originalWrite(ctx context.Context, out thrift.TProtocol, msg
 		requestBase = nil
 	}
 
-	// msg is void
-	if _, ok := msg.(descriptor.Void); ok {
+	// msg is void or nil
+	if _, ok := msg.(descriptor.Void); ok || msg == nil {
```

Comment:
It might be wrong since msg is of type `interface{}`, and therefore it will not be nil here

Reply:
Same for other nil check in this PR